lib/cldr/config/config.ex

```diff
@@ -2183,6 +2183,10 @@ defmodule Cldr.Config do
     config
   end
 
+  defp remove_gettext_only_locales(%{locales: :all} = config) do
```

Comment:
@kipcole9 While running test I encountered `:all - ["oc"]` below. Not sure if this has consequences.

Reply:
Yes, that certainly wouldn't work - will dig into this on the weekend